terratest/src/test/deploy_test.go

```diff
@@ -0,0 +1,160 @@
+package test
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	//k8sresource "k8s.io/apimachinery/pkg/api/resource"
+
+	"github.com/gruntwork-io/terratest/modules/helm"
+)
+
+const PegaHelmChartPath = "../../../charts/pega"
+
+// set action execute to install
+var options = &helm.Options{
+	SetValues: map[string]string{
+		"global.actions.execute": "deploy",
+	},
+}
+
+func TestPegaStandardTierDeployment(t *testing.T) {
+	t.Parallel()
+	// Path to the helm chart we will test
+	helmChartPath, err := filepath.Abs(PegaHelmChartPath)
+	require.NoError(t, err)
+
+	VerifyPegaStandardTierDeployment(t, helmChartPath, options, []string{"wait-for-pegasearch", "wait-for-cassandra"})
+}
+
+/*func TestPegaDeployments(t *testing.T) {
```

Comment:
Should these be disabled?  Maybe reenable or remove?

Reply:
Fixed